src/chains/supported-chains-list.ts

```diff
@@ -9,36 +9,39 @@ export const CHAINS = {
     OSMOSIS: "osmosis-4",
```

Comment:
should this file be in the s3 bucket? we don't want to have to bump the SDK every time we add a new chain

Reply:
It's a utility for developers (enumesque). Can't be fetched from s3 because static analysis would not work.
I don't have a better solution atm.